app/views/components/_share-links.html.erb

```diff
@@ -0,0 +1,28 @@
+<%
+  title ||= 'Share this page'
+  facebook_href ||= false
+  twitter_href ||= false
+%>
+<% if facebook_href || twitter_href %>
+<div class="app-c-share-links-wrapper">
+  <h2><%= title %></h2>
+
+  <div class="app-c-share-links" data-module="track-share-button-clicks">
+    <% if facebook_href %>
+      <%= link_to facebook_href,
+        target: "_blank",
+        rel: "noopener noreferrer",
+        class: "app-c-share-links__link js-share-facebook" do %>
+        <span class="app-c-share-links__icon app-c-share-links__icon--facebook"></span><span class="visually-hidden">Share on </span>Facebook<% end %>
```

Comment:
Turns out this is on one line to prevent whitespace problems - otherwise you end up with the problem @fofr noticed below, with slightly longer underlines on the text.